src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | null,
```

Comment:
Consider renaming `user` property to `assignedUser` for better clarity and consistency with `userId` property. 🤔